gravitee-io/gravitee-access-management
# feat(theme): manage security domain themes in the gateway

closes gravitee-io/issues#8125

**NOTE: Currently we can't change the color of the svg icons (eyes, arrows, ...)**

## :memo: Test scenarios 

1. Publish a theme with a custom color palette and a logo
2. Visit all the pages (login, reset password, registration, ...) and check if the theme has been applied

## :computer: Add screenshots for UI

<img width="1674" alt="theme-1" src="https://user-images.githubusercontent.com/1927438/185969162-e492bce3-0f85-4bf9-b95c-fbf8f6d085fc.png">
<img width="860" alt="theme-2" src="https://user-images.githubusercontent.com/1927438/185969178-a6052d63-8edf-4617-977f-ec8b30363301.png">
